qa/common/util.sh

```diff
@@ -213,6 +213,73 @@ function check_log_table_row_column() {
     return 0
 }
 
+# Check the table(s) by calling check_log_table_row_column and check_csv_table_row_column
+# See if the logic generalizes to the specify testing needs
+# Call check_log_table_row_column and check_csv_table_row_column directly if not
+function check_table_row_column() {
```

Comment:
Can you rename this to `check_analyzer_results`?

Reply:
Yes